templates/credentials/your-exams.html

```diff
@@ -34,7 +34,8 @@ <h1>
           <div class="p-notification__content">
             <h5 class="p-notification__title">Attention:</h5>
             <p class="p-notification__message">
-              Before start your exam you need to sign our <a href="/legal/confidentiality-agreement?tfa_7=cue%40canonical.com" target="_blank">confidentiality agreement.</a>
+              Before you start your exam you need to sign our <a href="/legal/confidentiality-agreement?tfa_7=cue%40canonical.com"
```

Comment:
Is this query `?tfa_7=cue%40canonical.com` necessary here?
```suggestion
              Before you start your exam you need to sign our <a href="/legal/confidentiality-agreement"
```